src/mono/msbuild/AndroidAppBuilder/Templates/AndroidManifest.xml

```diff
@@ -5,6 +5,8 @@
           a:versionName="1.0">
   <uses-sdk a:minSdkVersion="%MinSdkLevel%" />
   <uses-permission a:name="android.permission.INTERNET"/>
+  <uses-permission a:name="android.permission.READ_EXTERNAL_STORAGE"/>
+  <uses-permission a:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
nit: we shouldn't need external storage permission, we can use TMPDIR to store the results.

Reply:
I noticed xamarin.android guys use external storage for logs